src/preferences.py

```diff
@@ -7,50 +7,65 @@
 from graphs import file_io, graphs, misc, plot_styles, utilities
 
 
-class Preferences():
+class Preferences(dict):
```

Comment:
Took me a minute to find that Preferences now inherits from dict, which is why this all works. Clever solution.